app.py

```
@@ -1,88 +1,66 @@
-from datetime import datetime
-from flask import Flask, render_template, redirect
+from flask import Flask, render_template, request, redirect, url_for, flash
+from flask_sqlalchemy import SQLAlchemy
+from flask_wtf.csrf import CSRFProtect
 
 app = Flask(__name__)
-app.secret_key = secrets.token_urlsafe(32)
+# /// = relative path, //// = absolute path
+app.config['SQLALCHEMY_DATABASE_URI'] = 'sqlite:///db.sqlite'
+app.config['SQLALCHEMY_TRACK_MODIFICATIONS'] = False
+app.config['SECRET_KEY'] = 'dev-secret-key-change-in-production'
```

Ruff (0.15.6)

[error] Possible hardcoded password assigned to: "SECRET_KEY"

(S105)